csharp/src/Google.Protobuf/Google.Protobuf.nuspec

```diff
@@ -5,20 +5,34 @@
     <title>Google Protocol Buffers C#</title>
     <summary>C# runtime library for Protocol Buffers - Google's data interchange format.</summary>
     <description>See project site for more info.</description>
-    <version>3.0.0-beta*</version>
+    <version>3.0.0-alpha4</version>
```

Comment:
I am actually still not sure what the release version should be. I will try to talk to Jisi soon.